contracts/TokenHolderRevenueFund.sol

```diff
@@ -40,6 +41,15 @@ contract TokenHolderRevenueFund is Ownable, AccrualBeneficiary, Servable, Transf
     // -----------------------------------------------------------------------------------------------------------------
     string constant public CLOSE_ACCRUAL_PERIOD_ACTION = "close_accrual_period";
 
+    //
+    // Types
+    // -----------------------------------------------------------------------------------------------------------------
+    struct AccrualPeriod {
+        uint256 startBlock;
+        uint256 endBlock;
+        int256 amount;
```

Comment:
I feel some tension between the given name and the abstraction given by the implementation.
The name suggest abstraction of a time period which can be defined by startBlock and endBlock.
The actual implementation suggest abstraction of a relation between time period and  amount.
I am not sure what is intended?

Reply:
A tip: To always be consistent in range names and range logic, I have found the following name convention to be useful
   end - begin === 1
   last - first === 0
Granted, not all packages follow this convention and I find that regrettable.

Reply:
Good point. Only a late stage I added the `amount` property to the struct. I think a suitable name actually is `Accrual`, alternatively `PeriodicAccrual`.